libs/rays/include/rays/PathTracer.h

```diff
@@ -30,18 +30,32 @@ namespace filament {
 namespace rays {
 
 /**
- * Signals that a region of the generated image has become available, typically used for progress
- * notification. This can be called from any thread.
+ * RenderTarget denotes a client-owned image that the PathTracer renders into.
+ *
+ * All output planes must have 3 channels except AMBIENT_OCCLUSION, which must have 1.
+ */
+enum RenderTarget {
```

Comment:
I'm not sure RenderTarget is the best name here. I can't think of a better one though. BakeData? BakeTarget?

Reply:
These images can be written into for non-baking tasks as well (e.g. when invoking the pathtracer with SimpleCamera).  So I don't think bake should be in the name.  Maybe `OutputPlane`?

Reply:
Renamed to OutputPlane.